lib/puppet_x/templates/invoke_ps_command.erb

```diff
@@ -23,7 +215,10 @@ public class PuppetPSHost : PSHost
     public override Guid InstanceId { get { return _hostId; } }
     public override string Name { get { return "PuppetPSHost"; } }
     public override Version Version { get { return new Version(1, 0); } }
```

Comment:
I agree with @jpogran here, and I think that we should not be matching version to PS version, etc.  I've had to introspect hosts in utility code to detect features (i.e. WinRM doesn't do transcription) - so not mirroring the name / version of other hosts is the way to go IMHO.  I would say we should just use `base.Version`, except the field is `abstract` in [PSHost](https://msdn.microsoft.com/en-us/library/system.management.automation.host.pshost.version\(v=vs.85\).aspx).

I will say though that we should probably be bumping the version from the last `1.0` version in the module, to something newer like `1.1`, since we've added new features to the class (even though they're not shipped yet).